homeassistant/components/deconz/sensor.py

```diff
@@ -24,40 +18,47 @@
 
 async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
     """Old way of setting up deCONZ platforms."""
-    pass
 
 
 async def async_setup_entry(hass, config_entry, async_add_entities):
     """Set up the deCONZ sensors."""
     gateway = get_gateway_from_config_entry(hass, config_entry)
 
+    batteries = []
```

Comment:
Should this be a set if all items are unique?

Reply:
Absolutely. I will change that.